watertap/unit_models/ion_exchange_0D.py

```diff
@@ -287,51 +409,83 @@ def build(self):
 
-        self.p_drop_A = Var(
+        self.p_drop_A = Param(
             initialize=0.609,
+            mutable=True,
             units=pyunits.psi / pyunits.m,
             doc="Pressure drop equation intercept",
         )
 
-        self.p_drop_B = Var(
+        self.p_drop_B = Param(
             initialize=0.173,
+            mutable=True,
             units=(pyunits.psi * pyunits.hr) / pyunits.m**2,
             doc="Pressure drop equation B",
         )
 
-        self.p_drop_C = Var(
+        self.p_drop_C = Param(
             initialize=8.28e-4,
+            mutable=True,
             units=(pyunits.psi * pyunits.hr**2) / pyunits.m**3,
             doc="Pressure drop equation C",
         )
 
-        # ====== Resin variables ====== #
+        self.pump_efficiency = Param(
+            initialize=0.8,
+            mutable=True,
+            units=pyunits.dimensionless,
+            doc="Pump efficiency",
+        )
+
+        # Rinse, Regen, Backwashing params
 
-        self.resin_max_capacity = Var(
+        self.t_regen = Param(
+            initialize=1800,
+            mutable=True,
+            units=pyunits.s,
+            doc="Regeneration time",
+        )
+
+        self.rinse_bv = Param(
             initialize=5,
-            units=pyunits.mol / pyunits.kg,
-            bounds=(0, None),  # Perry's
-            doc="Resin max capacity",
+            mutable=True,
+            doc="Number of bed volumes for rinse step",
         )
 
-        self.resin_eq_capacity = Var(
-            initialize=1,
-            units=pyunits.mol / pyunits.kg,
-            bounds=(0, None),  # Perry's
-            doc="Resin equilibrium capacity",
+        self.bw_rate = Param(
+            initialize=5,
+            mutable=True,
+            units=pyunits.m / pyunits.hour,
+            doc="Backwash loading rate [m/hr]",
         )
 
-        self.resin_unused_capacity = Var(
+        self.t_bw = Param(
+            initialize=600,
+            mutable=True,
+            units=pyunits.s,
+            doc="Backwash time",
+        )
+
+        self.service_to_regen_flow_ratio = Param(
+            initialize=3,
+            mutable=True,
+            units=pyunits.dimensionless,
+            doc="Ratio of service flow rate to regeneration flow rate",
+        )
+
+        self.number_columns_redund = Param(
             initialize=1,
-            units=pyunits.mol / pyunits.kg,
-            bounds=(0, None),  # Perry's
-            doc="Resin available capacity",
+            mutable=True,
+            units=pyunits.dimensionless,
+            doc="Number of redundant columns for ion exchange process",
         )
+        # ==========VARIABLES==========
+        # COMMON TO LANGMUIR + FREUNDLICH
 
         self.resin_diam = Var(
             initialize=7e-4,
-            bounds=(5e-4, 1.5e-3),
-            units=pyunits.m,  # Perry's
+            bounds=(5e-4, 1.5e-3),  # Perry's
```

Comment:
FYI- @bknueven often suggests moving these inline comments above lines of code to prevent funkiness from black formatting.